# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
#feedback
int32 batteryState
int32 batteryMode
float64 procentSOC
float64 mAhSOC
